spreadsheet/update/update_query.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
+//
+// Use of this source code is governed by the terms of the Affero GNU General
+// Public License version 3.0 as published by the Free Software Foundation and
+// appearing in the file LICENSE included in the packaging of this file. A
+// commercial license can be purchased on https://unidoc.io.
+
+package update
+
+// Update types constants.
+const (
+	REMOVE_COLUMN byte = iota
+	REMOVE_ROW
```

Comment:
maybe skip the REMOVE_ROW as I believe that isnot supported yet?
Would make sense to add it if and when it is needed